src/js/domain.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+import { addHistory } from './history.js';
+
+export const setDomainEventListeners = (domain) => {
+  // domain delete tabs event
```

Comment:
これコメントがおかしい。
ここでやっているのはタブの並び替えだったはず。
```suggestion
  // sort tabs with URL
```

Reply:
これはただのミスだった。

popup.jsに移行した。